internal/server/v1/info/golden/variable_group_info/weather.json

```diff
@@ -0,0 +1,509 @@
+{
+  "node": "dc/g/Weather",
+  "info": {
+    "absoluteName": "Weather",
+    "childStatVars": [
+      {
+        "id": "ConsecutiveDryDays",
+        "searchNames": [
+          "Consecutive Dry Days of Place"
+        ],
+        "displayName": "Consecutive Dry Days",
+        "definition": "mp=consecutiveDryDays,pt=Place",
+        "hasData": true
+      },
+      {
```

Comment:
This is because of prod tables I guess?

Reply:
Yes, after using newest frequent, this file is much smaller.